### PR TITLE
5963 remove countdown

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#5963

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by Kaineatthelab** [comment]
Need to make sure the count down isn't running in the background and doesn't have any other consequences:)

File: client/src/components/payment/payment-modal.vue [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
       <v-card-title class="d-flex justify-space-between">
         <div>Confirm Name Request</div>
-        <countdown-timer :timerName="timerName" colorString="#003366" bgColorString="#efefef" style="float: right"/>
```
Comment (severinbeauvais) [inline client/src/components/payment/payment-modal.vue:7]: I suggest you add the close "X" here instead (as on the other similar modals). Check with Scott if needed.

File: client/src/components/payment/payment-modal.vue [hunk 2]
```diff
@@ -92,10 +85,6 @@ export default class PaymentModal extends Mixins(
   /** The model value for the dialog component. */
   private isVisible = false
 
-  private get timerName () {
-    return this.$PAYMENT_COMPLETION_TIMER_NAME
```
Comment (severinbeauvais) [inline client/src/components/payment/payment-modal.vue:96]: Can we get rid of all globals/properties that use this 'constant' also?

And also the configs?

Comment (vysakh-menon-aot) [reply client/src/components/payment/payment-modal.vue:96]: will do that

File: client/src/App.vue [hunk 3]
```diff
@@ -195,33 +193,6 @@ export default class App extends Mixins(SessionTimerMixin) {
...
-            await paymentModule.togglePaymentModal(false)
-            // Direct the user back to the start
-            await this.resetAppState()
-          },
-          timeoutMs: this.$PAYMENT_COMPLETION_TIMEOUT_MS
-        })
-      }
-    }
-  }
-
```
Comment (severinbeauvais) [inline client/src/App.vue:224]: Can TimerModal, onTimerModalExpired(), onTimerModalSessionExtended(), state stuff, etc also be deleted?

Or is this more than you want to do atm?

File: client/src/components/payment/payment-modal.vue (hunk 2 above)
Comment (severinbeauvais) [reply client/src/components/payment/payment-modal.vue:96]: And also PAYMENT_COMPLETION_TIMEOUT_MS I think.

**Review by severinbeauvais — APPROVED** [review]
Looks ok. Make sure you test well.

File: client/src/App.vue (hunk 3 above)
Comment (vysakh-menon-aot) [reply client/src/App.vue:224]: I dont think its related to payment. Its seems to be session timeout modal

Comment (severinbeauvais) [reply client/src/App.vue:224]: OK, can be cleaned up later.

File: client/public/config/configuration.json [hunk 4]
```diff
@@ -17,9 +17,6 @@
     "EXISTING_NR_TIMER_NAME": "existingNrTimer",
     "EXISTING_NR_TIMEOUT_MS": 0,
     "DEV_EXISTING_NR_TIMEOUT_MS": 0,
-    "PAYMENT_COMPLETION_TIMER_NAME": "paymentCompletionTimer",
-    "PAYMENT_COMPLETION_TIMEOUT_MS": 120000,
-    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 900000,
```
Comment (severinbeauvais) [inline client/public/config/configuration.json:22]: Excellent. Please add a note to the next Namerequest UI "release ticket" to delete these items from the config map (ie, when we deploy).

https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6065

**Review by severinbeauvais — APPROVED** [review]
Your updates look good.

**Review by lekshmimallika-aot — APPROVED** [review]

**Pull request merged**